### ISSUE TITLE
1.20.1 [The singularities have the same item ID's!!!]

### ISSUE THREAD

**Issue by TigerYoutubeGD** [xRemby/Avaritia-Universal#6]
I am trying to allow for automation of the crafting of things like the infinity ingot and catalysts... Those require the use of kubejs create which works well. The problem comes when singularities come into the picture... I have been trying to find the unique item id's of the singularities but had no luck... when I tried using the id that all of them used, it showed "uncraftable singularity" as a material needed... is there a way to change the current id's of the singularities or a way to manipulate the singularities folder to make unique id's?

**Comment by xRemby** [comment]
Yes, to delete all singularities, go to config/avaritia/singularities and delete all files. If you need to create your own, you can do it using notepad in the format.json, you can take a look at the iron singularity example.
